src/exports.c

```diff
@@ -385,28 +555,37 @@ exports_producer_init(config_setting_t *config)
     return exports;
 }
 
-bool
+int
```

Comment:
Some documentation on return values would be helpful (or maybe make return value a enum).

Reply:
True, especially because of json_c s weirdness.

Reply:
I pushed documentation and implemented boolean for filters.